packages/webapp/src/elections/api/eden-contract.ts

```diff
@@ -181,12 +199,16 @@ export const getParticipantsInCompletedRound = async (
         .signedBinaryToDecimal(bytes)
         .toString();
 
-    return await getTableRows(CONTRACT_MEMBER_TABLE, {
-        index_position: 2,
-        key_type: "i128",
-        lowerBound,
-        limit: 6,
-    });
+    const addWinner = isStillParticipating ? [member] : [];
+
+    return addWinner.concat(
+        await getTableRows(CONTRACT_MEMBER_TABLE, {
+            index_position: 2,
+            key_type: "i128",
+            lowerBound,
+            limit: 6,
```

Comment:
We're ending up with 7 results in our returned array no matter what the group size here. (6 limit plus winner). Options:

* Find a way to get the `upperBound` here.
* Have a sufficiently high limit and filter out those whose rep is not the winner. (but this would only work if there _is_ a winner, I believe).